src/Path/REPL/Command.hs

```diff
@@ -1,23 +1,20 @@
+{-# LANGUAGE TypeOperators #-}
 module Path.REPL.Command where
 
 import Path.Module
 import Path.Name
+import Path.Span (Spanned(..))
 import Path.Surface
-import Text.Trifecta.Rendering (Spanned(..))
+import Path.Term
 
 data Command
   = Quit
   | Help
-  | TypeOf (Spanned Surface)
-  | Decl (Spanned (Decl User (Spanned Surface)))
-  | Eval (Spanned Surface)
-  | Show Info
+  | TypeOf (Spanned (Term Surface User))
+  | Decl (Decl (Term Surface User))
+  | Eval (Spanned (Term Surface User))
+  | ShowModules
   | Reload
-  | Import (Spanned Import)
-  | Doc ModuleName
-  deriving (Eq, Ord, Show)
-
-data Info
-  = Bindings
-  | Modules
```

Comment:
Got rid of this because showing bindings properly is a bit tricky across the module graph.